HW1/Task4.py

```diff
@@ -0,0 +1,16 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое 
+# количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# 6 -> 1 4 1
+# 24 -> 4 16 4
+# 60 -> 10 40 10
+
+print('Введите количество журавликов, которое сделали дети:')
+s = int(input())
+
+if s % 6 == 0:
+    print('Петя сделал', s // 6, 'жур.')
+    print('Серёжа сделал', s // 6, 'жур.')
+    print('Катя сделала', (s // 6) * 4, 'жур.')
```

Comment:
Тут все верно, Марина, однако вы не могли не заметить, что происходит 4 раза выполнение одного и того же действия, а именно целочисленные деления на 6. Чтобы такого не происходило используют переменные. Можно даже в комбинации с моржовым оператором
```suggestion
    var = s // 6
    print('Петя сделал', var, 'жур.')
    print('Серёжа сделал', var, 'жур.')
    print('Катя сделала', var * 4, 'жур.')  # в первоначальном варианте тут скобки были не нужны.
```